deploy/WMAgent.production

```diff
@@ -10,12 +10,10 @@ COUCH_CERT_FILE=/data/certs/servicecert.pem
 COUCH_KEY_FILE=/data/certs/servicekey.pem
 GLOBAL_WORKQUEUE_URL=https://cmsweb.cern.ch/couchdb/workqueue
 WMSTATS_URL=https://cmsweb.cern.ch/couchdb/wmstats
-REQMGR_URL=https://cmsweb.cern.ch/reqmgr/rest
```

Comment:
Sorry if I have missed a change in the past that is explaining this already, but why do we need to remove this URL from the agent's configuration?

Reply:
It's been replaced by REQMGR2_URL (long time ago), this was a leftover.